tcms/testcases/tests/test_views.py

```diff
@@ -64,6 +65,36 @@ def test_user_in_default_group_sees_comments(self):
                 html=False
             )
 
+    def test_user_sees_bugs(self):
+        bug_1 = BugFactory(bug_id = 1)
+        bug_2 = BugFactory(bug_id = 2)
```

Comment:
The proper way is to modify the factory to contain 
`bug_id = factory.Sequence(lambda n: n)` and you can just create a BugFactory object without passing any parameters.